app/controllers/ProjectSidewalkAPIController.scala

```diff
@@ -418,6 +527,22 @@ class ProjectSidewalkAPIController @Inject()(implicit val env: Environment[User,
       }
       writer.close()
       Future.successful(Ok.sendFile(content = file, onClose = () => file.delete))
+    } else if (filetype != None && filetype.get == "shapefile"){
+      val streetBuffer: Buffer[StreetAttributeSignificance] = new ArrayBuffer[StreetAttributeSignificance]
+      for(streetAccessScore <- streetAccessScores){
+        streetBuffer.add(
+          new StreetAttributeSignificance(
```

Comment:
You should be able to remove any instances of the `new` keyword in Scala

Reply:
I have to use the `new` keyword to get an instance of ArrayBuffer and JTSCoordinate